packages/contracts-bedrock/scripts/invariant-doc-gen.ts

```diff
@@ -0,0 +1,168 @@
+import fs from 'fs'
+
```

Comment:
I'm fine with this not working on windows, but `path.join` could be useful to making this more portable. Everybody on the team develops with unix, altho there are some windows devs out there in the ethereum ecosystem